### ISSUE TITLE
OSM association with or agreement about openstreetmap.dk

### ISSUE THREAD

**Issue by mikini** [openstreetmap/operations#448]
Hi there.

I noticed that openstreetmap.dk (ccTLD for Denmark), which once redirected to openstreetmap.org, now has an A record to an IP (see below) which doesn't respond on port 80 or 443 (or ICMP or anything). According to [DK-Hostmaster](https://www.dk-hostmaster.dk/da/find-domaenenavn) the domain is registered to a Danish individual (I were actually the registrant before that, up to 2008). 

Looking up the A record IP (see below) reveals an association with Imperial College London, which to me indicates [some sort of connection with OSM](https://hardware.openstreetmap.org/thanks/#imperial-college-london) (I think I remember them being mentioned as hosting provider on the frontpage in the past). The name servers being at a well-known Danish free DNS service provider ([gratisdns.dk/Larsen Data](https://web.gratisdns.dk/)) suggests that DNS records are not maintained by OSM.

So my question; do, or did, OSMF/OWG have an agreement to serve/redirect users on openstreetmap.dk? Or were the redirect I previously experienced just someone independently having taken advantage of OSM servers redirecting graciously? I can see that they would do that today if the IP were correct (see below).

Are there any generic policies on ccTLDs associated with OSM? Could I suggest to the registrant to activate the redirect by replacing the A record with a CNAME to openstreetmap.org with your endorsement?
(alternatively, we might persuade dotsrc.org (provides DK based tile hosting of [rimfaxe](https://hardware.openstreetmap.org/servers/rimfaxe.openstreetmap.org/)) to host some community pages, I got them to set up a machine for [our import bot](https://wiki.openstreetmap.org/wiki/AutoAWS) but it ended up elsewhere). 

DNS and whois on A record:
```
$ dig +noall +answer any openstreetmap.dk
openstreetmap.dk.	43200	IN	SOA	ns1.gratisdns.dk. hostmaster.gratisdns.dk. 2017112201 10800 3600 2419000 43200
openstreetmap.dk.	43200	IN	NS	ns3.gratisdns.dk.
openstreetmap.dk.	43200	IN	NS	ns5.gratisdns.dk.
openstreetmap.dk.	43200	IN	NS	ns2.gratisdns.dk.
openstreetmap.dk.	43200	IN	NS	ns4.gratisdns.dk.
openstreetmap.dk.	43200	IN	NS	ns1.gratisdns.dk.
openstreetmap.dk.	1000	IN	A	193.63.75.99
$ whois 193.63.75.99|grep desc
descr:          Imperial College of Science, Technology and Medicine
descr:          JANET
descr:          JANET(UK) Network Operations Centre
descr:          Third floor
descr:          15 Fetter Lane
descr:          London EC4A 1BW
descr:          United Kingdom
$
```

Current servers asked to serve openstreetmap.dk on 80 & 443:
```
$ echo -ne "GET / HTTP/1.1\r\nHost:www.openstreetmap.dk\r\n\r\n"| nc openstreetmap.org 80 | grep -e ^HTTP/ -e ^Location:
HTTP/1.1 301 Moved Permanently
Location: https://www.openstreetmap.org/
$ echo -ne "GET / HTTP/1.1\r\nHost:www.openstreetmap.dk\r\n\r\n"| openssl s_client -quiet -servername www.openstreetmap.dk -connect openstreetmap.org:443 2>/dev/null | grep -e ^HTTP/ -e ^Location:
HTTP/1.1 301 Moved Permanently
Location: https://www.openstreetmap.org/
$ 
```

**Comment by Firefishy** [comment]
The redirect is generic provided for all sites which hit us with an unmatched host. I look after the OSMF/OSM.org owned domains, I cannot find any communications about openstreetmap.dk 

OSMF have registered trademarks to protect the OpenStreetMap name and brand.

We have openstreetmap country TLDs / GTLDs which we manage the registration / renewal and point to the local community sites / services.

**Comment by tomhughes** [comment]
I really don't understand what the question is here. I mean I do understand but I don't understand why you are asking operations about it - we just look after technical details, we don't get into making agreements with people, which appears to be the question here.

It would probably be clearer if you explained what your goal was rather than focusing a lot of largely irrelevant technical detail.

If you're looking to establish a Danish community site and use the name for that then I suggest you read up on the [trademark policy](https://wiki.osmfoundation.org/wiki/Trademark_Policy) and consult with [LWG](https://wiki.osmfoundation.org/wiki/Licensing_Working_Group) and/or [LCWG](https://wiki.osmfoundation.org/wiki/Local_Chapters_and_Communities_Working_Group) as appropriate.

**Comment by grischard** [comment]
> So my question; do, or did, OSMF/OWG have an agreement to serve/redirect users on openstreetmap.dk? Or were the redirect I previously experienced just someone independently having taken advantage of OSM servers redirecting graciously?

I have no record of an agreement. For openstreetmap.lu, I just set up a redirect myself and never asked anyone.

The OSMF would be happy to pay for the yearly fees if the registrar wanted to transfer the domain to us. We would then set up a redirect if that's desired, or make it available for community projects.

**Comment by mikini** [comment]
Thanks for the info @Firefishy & @grischard, and sorry @tomhughes if me dabbling in technicalities and policy caused the intention to be unclear.

I just wanted to scratch the itch that I saw openstreetmap.dk not redirecting. Nothing more.

Thinking that DNS might just have been neglected after [servers were moved to Amsterdam](https://blog.openstreetmap.org/2018/07/25/server-moves-imperial-equinix/) made it an operations question.

The slew of other questions were just my curiosity acting out, sorry.
